music21/musicxml/xmlToM21.py

```diff
@@ -4060,7 +4060,13 @@ def xmlOrnamentToExpression(
         self.setPlacement(mxObj, orn)
         return orn
 
-    def xmlDirectionTypeToSpanners(self, mxObj):
+    def xmlDirectionTypeToSpanners(
+        self,
+        mxObj: ET.Element,
+        mxDirection: ET.Element|None = None,
+        staffKey: int|None = None,
+        totalOffset: OffsetQL|None = None
```

Comment:
You've added three (optional) arguments to the method but they don't appear in the docs.  They should be explained with a demo.

Reply:
Btw -- seeing the parallel to the arguments in `setDynamicsDirection` the addition of these arguments is approved just so long as they're documented here (or if they're properly documented in setDynamicsDirection, you can just say that they have the same role as they have there...)

Reply:
I'll document them.

Reply:
Done.